frontend/lib/core/service/validation_service.dart

```diff
@@ -0,0 +1,20 @@
+class ValidationService {
+  static bool username(String text) {
+    return text.length > 1;
+  }
+
+  static bool email(String text) {
+    bool emailValid = RegExp(
+            r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+")
```

Comment:
hmm, its acceptable here due to situation we will not use it anywhere else in the code (i think, and i hope so)